reference/configuration/framework.rst

```diff
@@ -1704,12 +1704,15 @@ The possible values for this option are:
 cookie_secure
 .............
 
-**type**: ``boolean`` or ``'auto'`` **default**: ``'auto'``
+**type**: ``boolean`` or ``'auto'`` **default**: ``null``
 
 This determines whether cookies should only be sent over secure connections. In
 addition to ``true`` and ``false``, there's a special ``'auto'`` value that
 means ``true`` for HTTPS requests and ``false`` for HTTP requests.
 
+By default, it will use the value defined in the ``php.ini`` with the
+``session.cookie_secure`` directive.
```

Comment:
"By default" because value is null isn't it ?

Maybe something like this is better "When value is ``null``, ..." ?

Reply:
It is not even `null` by default: it is left unset; so I copy-pasted this from https://github.com/symfony/symfony-docs/blob/ab0b2da4fd5ffdb38ace52663aa6c8b5c7f4782a/reference/configuration/framework.rst?plain=1#L1589-L1596

which follows the same pattern.

I think it will require another PR because this is confusing, and some options don’t follow the same pattern like `cookie_path` which is also left unset by default to use php.ini’s `session.cookie_path`.

Reply:
Do you want to tackle the whole topic in this PR?

Reply:
I might as well :+1: